mappings/net/minecraft/entity/vehicle/AbstractMinecartEntity.mapping

```diff
@@ -25,7 +29,14 @@ CLASS axm net/minecraft/entity/vehicle/AbstractMinecartEntity
 		ARG 7 type
 	METHOD a setCustomBlockPresent (Z)V
 	METHOD b setCustomBlock (Lbvk;)V
+	METHOD c setDamageWobbleTicks (I)V
+		ARG 1 wobbleTicks
+	METHOD d setDamageWobbleSide (I)V
+		ARG 1 wobbleSide
 	METHOD k setCustomBlockOffset (I)V
+	METHOD l getDamageWobbleStrength ()F
```

Comment:
What's the wobble strength? Is it an angle? If yes, then should it be `getDamageWobbleAngle`?

Reply:
I think `getDamageWobbleAngle` would be wrong. That's calculated in the renderer using this and `getDamageWobbleTicks()`. Really, there should be `damageWobble`, `lastDamageWobble`, and `interpolateDamageWobble`, like there are for bubble columns, but Mojang appears to have taken a shortcut. If you want to make it clear it's an angle, it should be `getDamageWobbleMaxAngle`, but then we also need `getBubbleWobbleStrength` ~> `getBubbleWobbleMaxAngle` etc.